### PR TITLE
Add detectable prefix to ReactElement proptype warning.

### PR DESCRIPTION
**Pull request by jsfb**
Add detectable prefix to ReactElement proptype warning.

### CODE REVIEW

**Comment by jsfb** [comment]
cc @zpao

File: src/classic/element/ReactElementValidator.js [hunk 1]
```diff
@@ -268,8 +268,9 @@ function checkPropTypes(componentName, propTypes, props, location) {
         // same error.
         loggedTypeFailures[error.message] = true;
 
+        var preface = 'Failed ReactElement proptype check. ';
         var addendum = getDeclarationErrorAddendum(this);
-        warning(false, error.message + addendum);
+        warning(false, preface+error.message + addendum);
```
Comment (sebmarkbage) [inline src/classic/element/ReactElementValidator.js]: Spaces around operators. This is failing lint.

File: src/classic/element/ReactElementValidator.js [hunk 2]
```diff
@@ -268,8 +268,9 @@ function checkPropTypes(componentName, propTypes, props, location) {
         // same error.
         loggedTypeFailures[error.message] = true;
 
+        var preface = 'Failed ReactElement proptype check. ';
```
Comment (zpao) [inline src/classic/element/ReactElementValidator.js]: Should we say `propType` to match how it looks in code?

**Comment by sebmarkbage** [comment]
Would it be possible to fix the prefix? So that the one we use is the one with the prefix. "Failed ReactElement propType check" is pretty obscure for the open-source release.

**Comment by jsfb** [comment]
Fix the prefix in what way?  Specifically, what would you like the prefix to be?

**Comment by jsfb** [comment]
@sebmarkbage see fancy new prefix

**Pull request merged**